db-es-audit/src/find_granule_counts.py

```
@@ -0,0 +1,607 @@
+        s3_client - S3 client to save off the list of providers
+    
+    Returns:
+        A list of providers that have granules
+
+    Exceptions:
+        Logs error and exits on issue connecting to or querying the DB
+    """
+        
+    search_query = f"""
+    select table_name from all_tables where table_name like '%_GRANULES'
+    """
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)